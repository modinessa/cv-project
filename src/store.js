import { configureStore } from "@reduxjs/toolkit";
import { navigationPanelReducer } from "./components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationPanelReducer,
	}
});
