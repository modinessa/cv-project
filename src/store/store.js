import { configureStore } from "@reduxjs/toolkit";
import navigationReducer from "../components/NavigationPanel/navigationPanelReducer";

export const store = configureStore({
	reducer: {
		navigation: navigationReducer,
	}
});