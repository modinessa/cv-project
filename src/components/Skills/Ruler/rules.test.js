import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import { store } from '../../../store/store';
import { Ruler } from './index';

describe('Ruler', () => {
	it('renders component', () => {
		expect(renderer.create(
			<Provider store={store}>
				<Ruler/>
			</Provider>
		)).toMatchSnapshot();
	})
})