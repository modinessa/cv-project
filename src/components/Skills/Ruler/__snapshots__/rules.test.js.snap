// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ruler renders component 1`] = `
<div>
  <div
    className="ruler-container"
  >
    <div
      className="mark-25"
    />
    <div
      className="mark-50"
    />
    <div
      className="mark-75"
    />
  </div>
  <div
    className="ruler-labels"
  >
    <div
      className="item-1"
    >
      Beginner
    </div>
    <div
      className="item-2"
    >
      Proficient
    </div>
    <div
      className="item-3"
    >
      Expert
    </div>
    <div
      className="item-4"
    >
      Master
    </div>
  </div>
</div>
`;
